src/watcher.js

```diff
@@ -195,7 +198,46 @@ class Watcher extends SafeEventEmitter {
             })
         );
 
+        let twitchSendMessage;
+
+        const emitSendMessage = sendState => this.emit('chat.send_message', sendState);
```

Comment:
I honestly do not think this is the right solution to this problem. We need a separate module that functions as an ordered mutator for sending messages.